flask_stupe/pagination.py

```diff
@@ -1,17 +1,34 @@
 import functools
 
 from flask import request
-
 from flask_stupe import pymongo
 
 __all__ = []
 
 
 if pymongo:
-    def _paginate(cursor, skip=None, limit=None, sort=None, count=True):
-        metadata = getattr(request, "metadata", None)
-        if count and isinstance(metadata, dict):
-            metadata.update(count=cursor.count())
+
+    def _get_link_header(total_count, limit, skip):
+        template = '<{}?limit={}&skip={{skip}}>; ' \
+                   'rel="{{rel}}"'.format(request.base_url, limit)
```

Comment:
Not very important, but I feel it would be more readable in two instructions:
```python
        template = '<{}?limit={}&skip={{skip}}>; rel="{{rel}}"'
        template = template.format(request.base_url, limit)
```